content/klc/F4.3.adoc

```diff
@@ -6,6 +6,7 @@ Footprints that contain multiple pads or conductive elements that are *physicall
 
 . Multiple pads that are physically connected must share the same number
 . Thermal vias (if present) must share the same number as the thermal pad to which they are connected
-
+. Overlapping pads should be replaced with custom shapes
+. Pads and shapes should not be rotated arbitrarily. Ex: If you can swap width and height to avoid a 90deg rotation you should do so.
```

Comment:
The F4.3 rule heading is "Connected copper elements have the same pad number".
But the rotation-rule should also apply to any other pad in the footprint.

We could put that sentence into F6.3 and F7.4 (Pad requirements for ...)